packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/AvatarDemo/AvatarDemo.tsx

```diff
@@ -1,6 +1,6 @@
 import { Avatar, AvatarProps } from '@patternfly/react-core';
 import React, { Component } from 'react';
-import avatarImg from '../../../assets/images/imgAvatar.svg';
+const avatarImg = '../../../assets/images/imgAvatar.svg';
```

Comment:
Error was being thrown that the path could not be found.  Looking at `Avatar` component, the `src` prop where this is being used expects a URL string rather than an SVG file.